backends/exllamav2/model.py

```diff
@@ -163,30 +163,27 @@ def progress(loaded_modules: int, total_modules: int,
         if prompt_template_name:
             logger.info("Loading prompt template with name " f"{prompt_template_name}")
             # Read the template
-            self.prompt_template = get_template_from_file(prompt_template_name)
-        else:
-            # Then try finding the template from the tokenizer_config.json
-            self.prompt_template = get_template_from_model_json(
-                pathlib.Path(self.config.model_dir) / "tokenizer_config.json",
-                "chat_template",
-                "from_tokenizer_config",
-            )
+            try:
+                self.prompt_template = get_template_from_file(prompt_template_name)
+            except FileNotFoundError:
+                self.prompt_template = None
 
-            # Try finding the chat template from the model's config.json
-            # TODO: This may not even be used with huggingface models,
-            # mark for removal.
-            if self.prompt_template is None:
+            # Then try finding the template from the tokenizer_config.json
+            try:
```

Comment:
This implementation means I need to set a (dummy) `prompt_template` in the config even if it should use the template from tokenizer_config.json.

This is probably just wrong indendation?

Reply:
Oh, and even that does not work because it gets overridden in the next `try` block below that.

Reply:
Fixed in the latest commit.